e2e/jest.test-setup.ts

```diff
@@ -1,35 +1,37 @@
 const url = require('url');
-
 const userAgent = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.149 Safari/537.36';
 
-// Runs this beforeEach() before test's beforeEach().
-beforeEach(async () => {
+/**
+ * Before starting any test, set up page common properties:
+ * - Page view port
+ * - Page user-agent
+ * - Page navigation timeout
+ * - waitFor functions timeout
+ */
+beforeAll(async () => {
+  await page.setViewport({ width: 0, height: 0 });
   await page.setUserAgent(userAgent);
-  // Refer to https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagesetdefaultnavigationtimeouttimeout
+  // See https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagesetdefaultnavigationtimeouttimeout
   await page.setDefaultNavigationTimeout(90000);
   await page.setDefaultTimeout(60000);
 });
 
-// Runs this afterEach() before test's afterEach().
-afterEach(async () => {
-  await page.deleteCookie(...await page.cookies());
-  await jestPuppeteer.resetBrowser();
-});
-
-// Runs this beforeAll() before test's beforeAll().
-beforeAll(async () => {
+/**
+ * Before start of each test, enable network interception in new page and block unwanted requests.
+ */
+beforeEach(async () => {
   await page.setRequestInterception(true);
   page.on('request', async (request) => {
     const requestUrl = url.parse(request.url(), true);
     const host = requestUrl.hostname;
-    // to improve page load performance, block requests if it is not for application functions.
+    // to improve page load performance, block network requests unrelated to application.
     try {
       if (host === 'www.google-analytics.com'
-       || host === 'accounts.youtube.com'
-       || host === 'static.zdassets.com'
-       || host === 'play.google.com'
-       || request.url().endsWith('content-security-index-report')) {
-        await request.abort()
+         || host === 'accounts.youtube.com'
```

Comment:
I think you can use `fp.includes()` to see if the host in in one of the names you have here.

Reply:
That's a good tip. I'll test out in future PR.